https://bugs.gentoo.org/916995
https://github.com/WayfireWM/wf-config/pull/64
https://github.com/WayfireWM/wf-config/commit/af1bddc9d7191b9902edcb4c74572eac65577806

From af1bddc9d7191b9902edcb4c74572eac65577806 Mon Sep 17 00:00:00 2001
From: Kostadin <kocelfc@tutanota.com>
Date: Tue, 7 Nov 2023 16:41:55 +0200
Subject: [PATCH] Fix build with gcc 14 (#64)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix building with gcc 14 by adding #include <algorithm> to src/file.cpp

FAILED: libwf-config.so.0.9.0.p/src_file.cpp.o
ccache c++ -Ilibwf-config.so.0.9.0.p -I. -I.. -Iinclude -I../include -I/usr/include/libevdev-1.0 -I/usr/include/libxml2 -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c++17 -O0 -g -Wno-deprecated-declarations -fPIC -MD -MQ libwf-config.so.0.9.0.p/src_file.cpp.o -MF libwf-config.so.0.9.0.p/src_file.cpp.o.d -o libwf-config.so.0.9.0.p/src_file.cpp.o -c ../src/file.cpp ../src/file.cpp: In lambda function:
../src/file.cpp:419:25: error: ‘any_of’ is not a member of ‘std’
  419 |             return std::any_of(
      |                         ^~~~~~

Signed-off-by: Kostadin Shishmanov <kocelfc@tutanota.com>
--- a/src/file.cpp
+++ b/src/file.cpp
@@ -7,6 +7,7 @@
 #include <fstream>
 #include <cassert>
 #include <set>
+#include <algorithm>
 
 #include "option-impl.hpp"
 
